src/jit/compiler.h

```diff
@@ -134,6 +134,61 @@ const unsigned FLG_CCTOR = (CORINFO_FLG_CONSTRUCTOR | CORINFO_FLG_STATIC);
 const int BAD_STK_OFFS = 0xBAADF00D; // for LclVarDsc::lvStkOffs
 #endif
 
+//------------------------------------------------------------------------
+// HFA info shared by LclVarDsc and fgArgTabEntry
+//------------------------------------------------------------------------
+#ifdef FEATURE_HFA
+enum HfaElemKind : unsigned int
+{
+    HFA_ELEM_NONE,
+    HFA_ELEM_FLOAT,
+    HFA_ELEM_DOUBLE,
+    HFA_ELEM_SIMD16
```

Comment:
Shouldn't this be in a separate `HVA_ELEM` struct that also knows about `SIMD8`?

Reply:
(this may need to be expanded in the future for float16/float128)

Reply:
I'm not sure it's worth the cost of the distinction for `SIMD8` (or `FLOAT16`), as it would be a larger impact to the JIT/EE interface.

Reply:
I think this is probably fine for now, but it is probably worth distinguishing these long term as it may help diagnosing issues or bringing new platforms onboard.